public/style.css

```diff
@@ -9,3 +9,9 @@
     width: 200px;
 }
 
+<<<<<<< HEAD
```

Comment:
You still have MC code here showing the <<<<<<< and HEAD as well as >>>>>>> b5f14499875cebf7347086a30641b5a7c78081cb at the bottom.